framework/src/org/apache/cordova/engine/SystemWebChromeClient.java

```diff
@@ -155,7 +155,7 @@ public boolean onConsoleMessage(ConsoleMessage consoleMessage)
     {
         if (consoleMessage.message() != null)
             LOG.d(LOG_TAG, "%s: Line %d : %s" , consoleMessage.sourceId() , consoleMessage.lineNumber(), consoleMessage.message());
-         return super.onConsoleMessage(consoleMessage);
+        return super.onConsoleMessage(consoleMessage);
```

Comment:
This looks like a form of cleanup that I wish would be more clear in the description and in the commit message(s). Just a message like "other minor cleanup" would be awesome. +1 (+100) for making this kind of cleanup!

Reply:
Extracted the whitespace changes to a separate commit and updated the PR description.